src/main/java/net/pistonmaster/pistonmute/commands/MuteCommand.java

```diff
@@ -22,10 +21,28 @@ public final class MuteCommand implements CommandExecutor, TabExecutor {
     @Override
     public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
         if (args.length > 0) {
-            Player player = plugin.getServer().getPlayer(args[0]);
+            @Nullable UUID uuid = null;
+            @Nullable UUID senderUUID = null;
+            @Nullable String name = null;
 
-            if (player != null) {
-                if (player != sender) {
+            if (sender instanceof Player) {
+                senderUUID = ((Player) sender).getUniqueId();
+            }
+
+            if (args[0].matches("[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}")) {
```

Comment:
Replace this with a try catch of UUID#fromString() in a utility method. I don't like hardcoding this pattern.